app/models/review.rb

```diff
@@ -0,0 +1,13 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :film
+  has_many :comments, as: :commentable
+
+  validates :content, :film_id, :user_id, presence: true
+
+  def find_user
+    @user = User.find_by(id: self.user_id)
+    @user.email
```

Comment:
You should probably return the whole object and let the caller decide whether it just needs the email - this method is probably not needed anyway as you are just finding an object by id and reading an attribute. The call site can do that without introducing any indirection.